marathon-gradle-plugin/src/main/kotlin/com/malinskiy/marathon/MarathonRunTask.kt

```diff
@@ -31,14 +32,16 @@ open class MarathonRunTask : DefaultTask(), VerificationTask {
     lateinit var cnf: Configuration
     var ignoreFailure: Boolean = false
 
+    @OutputDirectory
+    lateinit var output: File
 
     @TaskAction
     fun runMarathon() {
         val instrumentationApk = testVariant.extractTestApplication()
         val applicationApk = applicationVariant.extractApplication()
 
         val baseOutputDir = if (extensionConfig.baseOutputDir != null) File(extensionConfig.baseOutputDir) else File(project.buildDir, "reports/marathon")
-        val output = File(baseOutputDir, flavorName)
+        output = File(baseOutputDir, flavorName)
```

Comment:
but this directory will be different for each gradle module/flavour, so it will not fix the problem.
the problem is in running tests for multiple modules/flavours at the same time.

for example, if we have 1 emulator and 2 gradle modules with tests. if we run `./gradlew marathon` it will try to run tests for both modules in parallel on the same emulator